microsoft-edge/devtools-guide-chromium/memory-problems/memory-101.md

```diff
@@ -23,216 +23,166 @@ ms.date: 07/21/2023
 
-Node `GC` dominates nodes `#1`, `#3`, and `#11`:
+<!-- ------------------------------ -->
+#### Distance
 
-![Node GC dominates nodes #1, #3, and #11](./memory-101-images/dominators-00.png)
- 
-Node `#3` is dominated by node `GC` and dominates node `#7`:
+The _distance_ of an object in the JavaScript heap is the number of nodes on the shortest path between the object and the GC root. The shorter the distance, the more likely it is that this object plays an important role in the memory usage of the webpage.
 
-![Node #3 is dominated by node GC and dominates node #7](./memory-101-images/dominators-01.png)
- 
-Node `#7` is dominated by node `#3` and dominates nodes `#8`, `#9`, and `#10`:
 
-![Node #7 is dominated by node #3 and dominates nodes #8, #9, and #10](./memory-101-images/dominators-02.png)
- 
-Node `#8` is dominated by node `#7` and doesn't dominate any nodes:
+<!-- ====================================================================== -->
+## Retainers
 
-![Node #8 is dominated by node #7 and doesn't dominate any nodes](./memory-101-images/dominators-03.png)
- 
-Node `#10` is dominated by node `#7` and doesn't dominate any nodes:
+An object's retainers are the others objects that hold references to it. The **Retainers** section of the **Memory** tool shows the objects that hold references to the object selected in the **Summary** view.
 
-![Node #10 is dominated by node #7 and doesn't dominate any nodes](./memory-101-images/dominators-04.png)
- 
-Node `#11` is dominated by node `#1` and doesn't dominate any nodes:
+The **Retainers** section of the **Memory** tool is sorted by distance by default, which means that the simplest retaining paths for an object are shown first.
 
-![Node #11 is dominated by node #1 and doesn't dominate any nodes](./memory-101-images/dominators-05.png)
+Any object with no retainers can be discarded by the browser’s garbage collector, which reduces memory usage.
 
 
 <!-- ====================================================================== -->
 ## V8 specifics
 
-When profiling memory, it is helpful to understand why heap snapshots look a certain way.  This section describes some memory-related topics specifically corresponding to the _V8 JavaScript virtual machine_ (abbreviated here as _V8 VM_, or just _VM_).
+When profiling memory, it's helpful to understand why heap snapshots look a certain way. This section describes how some objects are stored in memory by the _V8 JavaScript virtual machine_ (abbreviated here as _V8 VM_, or just _VM_), which can help you when analyzing heap snapshots in the **Memory** tool.
 
-### JavaScript object representation
+
+<!-- ------------------------------ -->
+#### JavaScript primitives
 
 In JavaScript, there are three primitive types:
 
 *  Numbers (such as `3.14159...`).
```

Comment:
@captainbrosset minor note: the string or value "3.14159..." with dots isn't a number type; it's not an actual example of a number value AFAIK.  removed dots.